AUTHORS.md

```diff
@@ -9,3 +9,7 @@ This is the official list of Fetch.ai authors for copyright purposes.
 * Oleg Panasevych <oleg.panasevych@n-cube.co.uk> [Panasevychol](https://github.com/panasevychol)
 * Peter Bukva <peter.bukva@fetch.ai> [pbukva](https://github.com/pbukva)
 * Yuri Turchenkov <yuri.turchenkov@fetch.ai> [solarw](https://github.com/solarw)
+
+## Other Contributors
+
+See the GitHub commit log for a list of recent contributors. We would like to thank everyone who has contributed to the project in any way.
```

Comment:
@5A11 @bryanchriswhite, my reasoning is only that we are thankful for all the contributors who have contributed to the project in any way.